docs/source/developer_guide/documentation.rst

```diff
@@ -79,16 +106,32 @@ Notes
 
 The notes section can include several types of information, including:
 
-- Mathematical details of a code object or other important implementation details
-- Links to alternative implementations of the code artifact that are external to ``sktime`` (e.g. the Java implementation of a sktime
-time series classifier)
+- Mathematical details of a code object or other important implementation details (using ..math or :math:`` functionality)
+- Links to alternative implementations of the code artifact that are external to ``sktime`` (e.g. the Java implementation of a sktime time series classifier)
 - state changing methods (sktime estimator classes)
 
 References
 ~~~~~~~~~~
 
 sktime estimators that implement a concrete algorithm should generally include citations to the original research article, textbook or other resource
-that describes the algorithm. Other code artifcations
+that describes the algorithm.
+
+Where applicable this should be done by providing a link between the description where the reference is applicable (say in the extended summary or notes)
+and the references section.
```

Comment:
```suggestion
```suggestion
This should be done by adding references into this references section, and then linking to these in other parts of the docstring.
```
Trying to make sentence clearer